src/task/script.js

```diff
@@ -303,3 +303,94 @@ function handleStatusChange(event = { target: { value: 'AVAILABLE' } }) {
 }
 
 document.getElementById('visibity-hidden').classList.remove('visibity-hidden');
+
+const assigneeEl = document.getElementById('assignee');
+
+function debounce(func, delay) {
+  let searchDelay;
+  return (...args) => {
+    if (searchDelay) {
+      clearTimeout(searchDelay);
+    }
+
+    searchDelay = setTimeout(() => {
+      func(...args);
+    }, delay);
+  };
+}
+
+async function autoComplete(searchInput) {
+  try {
+    const response = await fetch(`${API_BASE_URL}/members`);
+    const data = await response.json();
+    removeAssignee();
+    if (searchInput !== '') {
```

Comment:
Let's use **trim()** here since we use it before making a **POST** request

Reply:
Added trim method